client/src/pages/navigation.tsx

```diff
@@ -1,24 +1,33 @@
-import Image from "next/image";
-import Button from "../../components/Button";
-import Header from "../../components/Header";
-import DirectionsCard from "../../components/DirectionsCard";
-import groundFloor_main from "../../images/groundFloor_main.png";
+import Button from "../components/Button";
+import Header from "../components/Header";
+import DirectionsCard from "../components/DirectionsCard";
+import Map from "../components/Map";
+import { SubmapProviderImpl } from "../logic/impl/SubmapProviderImpl";
 
-export default function Navigation(){
+
+type Prop = {
+    submapId: number;
+}
+
+export default function Navigation({ submapId=3 }: Prop){
+
+    const submap = new SubmapProviderImpl();
+    const submapImage = submap.getSubmapImage(submapId);
```

Comment:
Resolve this comment after all others. Now we need to implement the Navigation component (i.e. navigation page) to be compatible with the rest of the app.

Navigation will receive data from the home page using Link and to=... (https://stackoverflow.com/questions/52238637/react-router-how-to-pass-data-between-pages-in-react) - fell free to suggest if there is something better.
Navigation will use the MapNavigator interface (mock it) to create a path to draw.

The navigation needs to receive needed parameters from the home page. Prop should be `startNodeId: number, endNodeFilter: MapNodeFilter` so the MapNavigator interface can be created and called in the Navigation page. Mock everything you need to get a (mocked) NavigationDirections object.

All of this is an initialization process for the navigation page. Your navigation page should then handle the NavigationDirections object. Draw the current step's path from it. (test path being on multiple submaps - but each navigation step should be only on one submap, of course).

Remove the submapId from the prop. You will find a submapId in the NavigationDirections.

As you already know, the Map component shouldn't know about anything of this. It just receives a path to draw and a map on which to draw that path.

After this, we will have a fully functional navigation page 😃